src/components/core/DpDashboardTaskCard.vue

```diff
@@ -75,7 +90,7 @@ export default {
     }
 
     // Get count of segments assigned to the current user
-    const segmentUrl = Routing.generate('api_resource_list', { resourceType: 'StatementSegment' })
+    const segmentUrl = Routing.generate(this.resourceListRoute, { resourceType: 'StatementSegment' })
```

Comment:
see above and all other changes

Reply:
See:
https://github.com/demos-europe/demosplan-ui/pull/91#discussion_r1093328892
https://github.com/demos-europe/demosplan-ui/pull/91#discussion_r1093359398